### PR TITLE
Fixes flagged by loc

### PR DESCRIPTION
**Pull request by shaynacohen**
SHOT22-199, SHOT22-271, SHOT22-311, SHOT22-158, SHOT22-156, SHOT22-155, SHOT22-154, SHOT22-308, SHOT22-200, SHOT22-203, SHOT22-219, SHOT22-274, SHOT22-275, SHOT22-309, SHOT22-312, SHOT22-319, SHOT22-322

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/2e958f95858081d2faed48192763a9df66918236/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/d4caa36b45c06cda1a63cc589e19aa455ce0f295/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/46b2520581ce9b196f38a4e888999c0d0d3e1a58/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/94aad223a8e6154c7b7cc9faf3571a772567405a/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/2b55441025a41a7162a98035b9b68c9a8f8e03a8/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/89b912af42a352c7110e6c7724d91e1427ca5cd0/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/857f5a18abb2b8d71f31a38c809a203999a7834d/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/1fd6e9adcb0594e8f9d00b030877dba028d0b323/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/56337161f8e005026fed12ab9deedf27ca83aacc/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/a818643cc17b8840540abce7486f2daf8c4f808a/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/473da6a4f3fa2284e99dedcd92b0bd2b7b5cda80/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/259f09d1b83f28131c2a5c4f4add08eeb2ce89d6/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/a4508a571a6a92fd983b566772fa99d13cc781ee/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/c20b480126eea7789b5ca343223a4feba2d3dd23/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/09b80384231101c7465ffa430161a62fa33c7b87/index.html)

File: docs/en/guides/pipeline-integrations/getting-started/editing_app_setting.md [hunk 1]
```diff
@@ -322,9 +322,9 @@ By selecting **+New File**, you began to work on a new asset and the `asset_step
 
 ![Art asset env](./images/editing_app_setting/23_Art_Asset.png)
 
-**Art, Asset** tells you’re working on and what environment you’re in.
+**Art, Asset** > Text Overlay tells you’re working on and what environment you’re in.
```
Comment (drewavisautodesk) [inline docs/en/guides/pipeline-integrations/getting-started/editing_app_setting.md:325]: Couple of curly quotes here

File: docs/en/guides/pipeline-integrations/getting-started/editing_app_setting.md [hunk 2]
```diff
@@ -322,9 +322,9 @@ By selecting **+New File**, you began to work on a new asset and the `asset_step
...
 
-**Step 27:** Select **Art, Asset > Work Area Info…** to display what the parameters are in your current work area. 
+**Step 27:** Select **Art, Asset Text Overlay > Work Area Info…** to display what the parameters are in your current work area.
```
Comment (drewavisautodesk) [inline docs/en/guides/pipeline-integrations/getting-started/editing_app_setting.md:327]: The ... looks like a single UTF-8 char here

File: docs/en/reference/pipeline-integrations/toolkit-apps/tk-multi-launchapp.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ launch_nuke:
 
 Applications often require certain certain environment variables, plugin paths, etc. to be set in order to properly function in a studio pipeline. The launch app helps cover this case via the 'before_app_launch' hook, which allows you to define a snippet of code to run each time an application starts up. By default, the 'before_app_launch' hook is a simple pass-through that does nothing, but it can be overridden by following the instructions in <a href='https://developer.shotgridsoftware.com/425b1da4/#hooks'>this doc</a>.
 
-For example, if you use <a href='https://www.zyncrender.com/'>Zync Render</a>, the Zync Maya plugin directory needs to be part of both $PYTHONPATH and $XBMLANGPATH. To have the launch app set these environment variables, update the 'before_app_launch' hook with a few lines of code, as follows:
+For example, if you use Zync Render, the Zync Maya plugin directory needs to be part of both $PYTHONPATH and $XBMLANGPATH. To have the launch app set these environment variables, update the 'before_app_launch' hook with a few lines of code, as follows:
```
Comment (drewavisautodesk) [inline docs/en/reference/pipeline-integrations/toolkit-apps/tk-multi-launchapp.md:61]: Might want to put these envars in ` ` codephrases.

**Review by drewavisautodesk — APPROVED** [review]

File: docs/en/guides/pipeline-integrations/getting-started/editing_app_setting.md (hunk 2 above)
Comment (shaynacohen) [reply docs/en/guides/pipeline-integrations/getting-started/editing_app_setting.md:327]: Fixed, thanks for flagging!

File: docs/en/reference/pipeline-integrations/toolkit-apps/tk-multi-launchapp.md (hunk 3 above)
Comment (shaynacohen) [reply docs/en/reference/pipeline-integrations/toolkit-apps/tk-multi-launchapp.md:61]: Fixed, thanks for flagging!

File: docs/en/guides/pipeline-integrations/getting-started/editing_app_setting.md (hunk 1 above)
Comment (shaynacohen) [reply docs/en/guides/pipeline-integrations/getting-started/editing_app_setting.md]: Fixed, thanks for flagging!

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/1f9be5a080107600d5d30ef1329672cfb0ea750d/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/639e79313ef58ffc58c202941726be6141d7bd53/index.html)

**Pull request merged**